recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python setup.py install --old-and-unmanageable
```

Comment:
I'm happy with using `--single-version-externally-managed --record record.txt`. Is there some reason to use `--old-and-unmanageable`? I can't seem to find docs on it.